lib/vsc/mympirun/pmi/pmi.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2019-2019 Ghent University
+#
+# This file is part of vsc-mympirun,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-mympirun
+#
+# vsc-mympirun is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# vsc-mympirun is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with vsc-mympirun.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+PMI instances to track version and flavours etc etc
+"""
+
+PMIX = 'pmix'
+PMI = 'pmi'
+
+class PMI(object):
+    FLAVOUR = None
+    VERSION = None
+
+    def __str__(self):
+        return "PMI %s version %s" % (self.FLAVOUR, self.VERSION)
+
+class PMIv2(object):
```

Comment:
Why does this derive from `object`? Why not have the `__str__` method defined here?

Reply:
fixed